runner/runner.go

```diff
@@ -100,12 +99,11 @@ func (r *Runner) StartAll(serviceErrCh chan error) ([]topological.Task, error) {
 
 func (r *Runner) StopAll() (map[string]*os.ProcessState, error) {
 	tasks := allTasks(r.serviceInstances, func(ctx context.Context, service *ServiceInstance) error {
-		if service.Type == "group" {
+		if service.Type == "group" || service.Type == "task" {
```

Comment:
Wouldn't this orphan a long-running task when StopAll is called? I'm guessing you did this because tasks don't have the default-signal logic which got tripped up because you're passing `nil`?